config/builder-config.yml

```diff
@@ -4,27 +4,27 @@ dist:
     description: "Custom Lightstep Partner OpenTelemetry Collector distribution" # a long name for the application. Optional.
     output_path: build # the path to write the output (sources and binary). Optional.
     version: "${TAG}" # Builder config gets overriden by Docker build.
-    otelcol_version: "0.69.1"
+    otelcol_version: "0.78.1"
 
     # the version for your custom OpenTelemetry Collector. Optional. NOTE - this will be overriden by CI/CD during deployment.
 
 receivers:
-  - gomod: "github.com/lightstep/telemetry-generator/generatorreceiver v0.11.9"
+  - gomod: "github.com/lightstep/telemetry-generator/generatorreceiver v0.12.0"
     name: "generatorreceiver"
     path: "./generatorreceiver"
-  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver v0.69.0"
+  - gomod: "github.com/open-telemetry/opentelemetry-collector-contrib/receiver/prometheusreceiver v0.78.0"
```

Comment:
non blocking question: Curious why this is `v0.78.0` instead of `v0.78.1`. Same question for `spanmetricsprocessor` and `attributesprocessor`

Reply:
The contrib repo didn't release a .1 patch release right away. For the collector itself, the only component for which the v0.78.x differences matter is the batchprocessor.